samples/templates/default-azuredeploy.json

```diff
@@ -158,7 +167,10 @@
             "FhirServer:Security:Authentication:Authority": "[parameters('securityAuthenticationAuthority')]",
             "FhirServer:Security:Authentication:Audience": "[parameters('securityAuthenticationAudience')]",
             "CosmosDb:ContinuationTokenSizeLimitInKb": "1",
-            "DataStore": "CosmosDb"
+            "DataStore": "CosmosDb",
```

Comment:
What happens with the SQL template deployment?

Reply:
I haven't added these changes this to the SQL template since we don't yet support export for SQL.

Reply:
Update the sql template also with export options since we know support export in SQL also!